perllib/FixMyStreet/Script/UpdateAllReports.pm

```diff
@@ -271,6 +272,7 @@ sub calculate_top_five_bodies {
 
     my $bodies = FixMyStreet::DB->resultset('Body')->search;
     while (my $body = $bodies->next) {
+        next if $body->name eq 'Thamesmead';
```

Comment:
Would be better to exclude this from the `search` above, I guess, rather than skipping here.